src/Lodge/Postcode/Postcode.php

```diff
@@ -136,4 +129,25 @@ public function lookup($postcode)
 		return $array;
 	}
 
+    private function callGoogleApi($url) 
+    {
+        $url = $this->addApiKeyToUrl($url);
+        $json = json_decode(file_get_contents($url));
```

Comment:
Can we wrap a `try/catch` around the `file_get_contents`, that throws the `ServiceUnavailableException`